File: Source/OregonTilth.Web/src/app/shared/models/forms/field-labor-by-crop/field-labor-by-crop-create-dto.ts
import { FieldStandardTimeDto } from "../../generated/field-standard-time-dto";
import { FieldStandardTimeSummaryDto } from "../field-standard-times/field-standard-time-summary-dto";

export class FieldLaborByCropCreateDto {
    WorkbookID: number;
    CropID: number;
    FieldStandardTimes: FieldStandardTimeSummaryDto[];
    FieldStandardTimeID: number;
    Notes: string;
    constructor(obj?: any) {
        Object.assign(this, obj);
    }
}

File: Source/OregonTilth.Models/DataTransferObjects/Workbook/Forms/FieldLaborByCrop/FieldLaborByCropCreateDto.cs
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;

namespace OregonTilth.Models.DataTransferObjects
{
    public class FieldLaborByCropCreateDto
    {
        [Required] public int WorkbookID { get; set; }
        [Required] public int CropID { get; set; }
        [Required] public List<FieldStandardTimeDto> FieldStandardTimes { get; set; }
        [MaxLength(2000)] public string Notes { get; set; }
    }
}